testing/btest/cluster/websocket/zeromq/manager-worker-ping-pong.zeek

```diff
@@ -0,0 +1,148 @@
+# @TEST-DOC: Test WebSockets clients when the ZeroMQ cluster backend is enabled.
```

Comment:
If you add a bit more detail here then I don't have to piece together from code what sort of ping/pong protocol you have implemented across three components.

Reply:
Added a few notes about the sceanrio.